### PR TITLE
feat(graphql-api): enhance must-change-password script to accept a plain text

### PR DESCRIPTION
**Pull request by Cronus1007**
## Because
- 

## This pull request
-

## Issue that this pull request solves

Closes: #5895 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of change in user interface.

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

**Comment by Cronus1007** [comment]
@chenba Awaiting for your review?

File: packages/fxa-graphql-api/src/scripts/must-change-password.ts [hunk 1]
```diff
@@ -36,21 +36,64 @@ const logger = mozlog(config.log)('must-change-password');
...
+} else if (program.uids) {
+  uids = getItems('uids');
+}
+
+if (!emails.length && !uids.length) {
+  console.error('No `emails` or `uids` found inside the specified file');
+  process.exit(1);
+}
+
+function getItems(type) {
```
Comment (dannycoates) [inline packages/fxa-graphql-api/src/scripts/must-change-password.ts:73]: It doesn't look like `type` is used, should it be?

**Review by dannycoates — APPROVED** [review]
Thanks @Cronus1007 this looks good to me

File: packages/fxa-graphql-api/src/scripts/must-change-password.ts (hunk 1 above)
Comment (Cronus1007) [reply packages/fxa-graphql-api/src/scripts/must-change-password.ts:73]: @dannycoates I agree that it is safe to remove type from both the files `must-enhance-password.ts` and `must-reset.js`

Comment (Cronus1007) [reply packages/fxa-graphql-api/src/scripts/must-change-password.ts]: Shall I also rebase my commits.

Comment (dannycoates) [reply packages/fxa-graphql-api/src/scripts/must-change-password.ts]: > One more thing that there is no such test file for `must-enhance-password.ts` so shall I make a new issue for this.

sure, you're welcome to :)

Comment (Cronus1007) [reply packages/fxa-graphql-api/src/scripts/must-change-password.ts]: @dannycoates So is the pr ready to be merged?

**Pull request merged**